graph_model/trie_model/trie.py

```diff
@@ -29,4 +30,20 @@ def autocomplete(self, prefix, top_n):
         # sort output titles by score
         result.sort(key=lambda item: item['score'], reverse=True)
         print('Normalx search ->\nConsidered: ', len(result), result[:3], '\nPrefix: ',prefix)
-        return result[:top_n]
\ No newline at end of file
+        return result[:top_n]
+
+    def to_dict(self):
```

Comment:
```suggestion
    def save(self, fileobj:TextIO):
```
Please save and load to a fileobject.